MidCamp/midcamp
# Update Webform, create Session Feedback survey

# Description

- Updates Webform module to 5.1.0
- Exports updated config related to the changes in Webform
- Adds a Likert Five Scale option for rating session on a 1-5 scale.
- Creates a basic session feedback form
- Adds a field to Topic proposals to assign the feedback form
- Creates action to assign the feedback form to topics
    - If an end time is present, the webform will be scheduled to open at that time
    - If an end time is not present, the webform will be open by default

# To Test

- Install updates with `composer install`
- Import config with `drush cim -y`
- Navigate to the content overview page.  Observe there is an "Assign session survey" action
- Use the action on a 2019 topic node.  Observe there is now a webform assigned to the node, scheduled to begin at the session end time
